.github/workflows/test.yml

```
@@ -4,14 +4,17 @@ on: [push, pull_request]
 
 jobs:
   unit:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node: [10, 12, 13, 14, 15]
     steps:
       - name: Checkout
         uses: actions/checkout@v1
```

```suggestion
        uses: actions/checkout@v2.3.4
```
updates to using environment files in the checkout action